docs/machine-learning/how-to-guides/serve-model-web-api-ml-net.md

```diff
@@ -101,7 +101,7 @@ You need to create some classes for your input data and predictions. Add a new c
 
 To make a single prediction, use [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602). In order to use [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) in your application you must create it when it's needed. In that case, a best practice to consider is dependency injection.
 
-The following link provides more information if you want to learn about [dependency injection in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.1).
+The following link provides more information if you want to learn about [dependency injection in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/dependency-injection).
```

Comment:
Not sure if ML.NET guide publishes an offline book to VS @natke, but if you do, then this kind of link makes the books in between books work.
 
```suggestion
The following link provides more information if you want to learn about [dependency injection in ASP.NET Core](/aspnet/core/fundamentals/dependency-injection).
```